lib/app/pages/source_page.dart

```diff
@@ -44,10 +44,10 @@ class SourcePage extends HookConsumerWidget {
       autofillHints: const [AutofillHints.url],
       required: true,
       validator: (value, label) {
-        if (Uri.tryParse(value!) == null) {
+        if (Uri.tryParse(value!)?.isAbsolute == false ||
+            !value.contains('http')) {
           return '$label must be a valid URL';
         }
-        return null;
```

Comment:
Looks like we need to keep this line though for the analyzer.

Reply:
@austinried should be fixed now